edunext_openedx_extensions/ednx_microsites/tests/test_change_domain.py

```diff
@@ -0,0 +1,33 @@
+"""This module include a class that checks the command change_domain.py"""
+
+from io import StringIO
+
+from django.test import TestCase
+from django.core.management import call_command
+
+from edunext_openedx_extensions.ednx_microsites.models import Microsite
+
+
+class ChangeDomainTestCase(TestCase):
+    """ This class checks the command change_domain.py"""
+
+    # pylint: disable=no-member
+    microsite = Microsite
+    # pylint: disable=invalid-name
+
+    def setUp(self):
+        """This method creates microsite objects in database"""
+
+        self.microsite.objects.create(subdomain="first-test-prod.edunext.io")
```

Comment:
I dont see the point in testing `"first-test-prod.edunext.io"` and `"second-test-prod.edunext.io"`. One of them should be `"first.test.prod.com"` or something like that, and then on the asserts we make sure that: `"first.test.prod.com"` does not exist, but `"first-test-prod-com-stage.edunext.co"` does.